Emby.Dlna/PlayTo/PlayToController.cs

```diff
@@ -889,6 +889,11 @@ public Task SendMessage<T>(SessionMessageType name, Guid messageId, T data, Canc
             return Task.CompletedTask;
         }
 
+        public void CloseAllWebSockets(CancellationToken cancellationToken)
+        {
+            throw new NotImplementedException();
```

Comment:
I think it's fine to just make this function do nothing rather then throwing an exception

Reply:
Done!